ballerina/segment_group_reader.bal

```diff
@@ -32,8 +31,8 @@ isolated function readSegmentGroup(EdiUnitSchema[] currentUnitSchema, EdiContext
             return error Error("Segment schema cannot be empty.");
         }
         string sDesc = context.ediText[context.rawIndex];
-        string segmentDesc = regex:replaceAll(sDesc, "\n", "");
-        
+        string:RegExp newline = re `\n`;
```

Comment:
Lets move this regex to constants file. Seems this is reused a lot.

Reply:
Add it as a util function with [d88fbcd](https://github.com/ballerina-platform/module-ballerina-edi/pull/30/commits/d88fbcd7000220b59e8c7432406c9f427321a576). Is that okay?